### ISSUE TITLE
`Drive.get_item_by_path` does not initiate `_parent` attribute

### ISSUE THREAD

**Issue by MattSom** [O365/python-o365#643]
`Folder.upload_file` returns a `File` object (`DriveItem`) which has a `_parent` attribute loaded with the `Folder` object. However after you get this item by its path on the drive with `Drive.get_item_by_path`, this connection is not getting initialized, `_parent` field is `None`.

All in all directory handling logic is missing from this repository. 
The models are quite smart only there's a lot of practically needed functionality which is missing. I had to implement them in my layer, while they should have been this repo's job.

**Comment by janscas** [comment]
Yes, a lot of work could (and should) have been done to improve this library... but as hard as it sounds to you that requires time and effort which I don't have.

If you need something, instead of complaining, just submit PR.

**Comment by MattSom** [comment]
> If you need something, instead of complaining, just submit PR.

It is called feedback.
